src/app/pages/BuyPage/components/BuyFormContainer/index.module.scss

```diff
@@ -0,0 +1,82 @@
+// swap btn
+.swap-btn-container {
```

Comment:
Please update the classes to use `tailwind CSS`  amap. for instance, 
`@apply tw-m-auto;`
Refer the line 7.

Reply:
fixed